### PR TITLE
add #about methods to checks

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: bin/check [hunk 1]
```diff
@@ -22,12 +22,20 @@ end.freeze
 # If there are no options given, invoke help
 ARGV << "-h" if ARGV.empty?
 
+BLURBS = ALL_CHECKS.map do |ch|
```
Comment (mcritchlow) [inline bin/check]: `BLURBS` could maybe be something like `CHECK_DESCRIPTIONS` or `DESCRIPTIONS`? I think for the `map` it reads fine, just in the `Available checks:` section `#{BLURBS}` seems _subjectively_ a bit more questionable.

**Review by mcritchlow — COMMENTED** [review]
👍 great idea.

I'm not necessarily advocating for this over what you've implemented, but part of me wonders about making the about methods entries in a yml config. Like say this whole concept took off and others wanted to use it, in like, other languages. Probably scope creep. Just putting it out there.

File: bin/check (hunk 1 above)
Comment (dunn) [reply bin/check]: fiiiine

**Comment by dunn** [comment]
Yeah the way it auto-detects new modules right now is pretty yikes, so something better that also allows plugins in any language would be very cool.

**Comment by dunn** [comment]
Tests are failing because the XSD things aren't downloading https://travis-ci.org/ucsblibrary/metadata-ci/builds/273018099#L596

**Pull request merged**